src/dotnet/commands/dotnet-restore3/Program.cs

```diff
@@ -97,6 +103,13 @@ public static int Run(string[] args)
                     msbuildArgs.Add($"/p:RestoreIgnoreFailedSources=true");
                 }
 
+                // By default restore is recursive, if set to false turn it off.
```

Comment:
Sounds good, I think all of the restore parameters need to be looked at again and updated to match the rest of the CLI commands. Ex: verbosity